addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/SqlInjectionScanRule.java

```diff
@@ -19,9 +19,6 @@
  */
 package org.zaproxy.zap.extension.ascanrules;
 
-import difflib.Delta;
-import difflib.DiffUtils;
-import difflib.Patch;
```

Comment:
I believe this is the [only place](https://github.com/search?q=repo%3Azaproxy%2Fzap-extensions%20path%3A%2F%5EaddOns%5C%2Fascanrules%5C%2F%2F%20difflib&type=code) it's used in ascanrules, remove the dependency from the build file?
https://github.com/zaproxy/zap-extensions/blob/564877892297f209ad9330c855e19aec07accee2/addOns/ascanrules/ascanrules.gradle.kts#L54

Reply:
Good spot - will do!

Reply:
Done